core/modules/user/user.admin.inc

```diff
@@ -150,23 +154,23 @@ function user_admin_settings($form, &$form_state) {
     '#default_value' => $config->get('user_email_match'),
   );
   module_load_include('inc', 'user', 'user.pages');
-  $form['registration_cancellation']['user_cancel_method'] = array(
-    '#type' => 'item',
-    '#title' => t('When cancelling a user account'),
-    '#description' => t('Users with the %select-cancel-method or %administer-users <a href="@permissions-url">permissions</a> can override this default method.', array('%select-cancel-method' => t('Select method for cancelling account'), '%administer-users' => t('Administer user accounts'), '@permissions-url' => url('admin/config/people/permissions'))),
-  );
-  $form['registration_cancellation']['user_cancel_method'] += user_cancel_methods();
+  $form['registration_cancellation'] += user_cancel_methods();
+  $form['registration_cancellation']['user_cancel_method']['#title'] = t('Default account cancellation method');
   foreach (element_children($form['registration_cancellation']['user_cancel_method']) as $element) {
-    // Remove all account cancellation methods that have #access defined, as
+    // Remove the individual options descriptions provided by
+    // user_cancel_methods() (these are only displayed on the confirmation form
+    // for own account cancellation).
+    unset($form['registration_cancellation']['user_cancel_method'][$element]['#description']);
+    // Disable all account cancellation methods that have #access defined, as
     // those cannot be configured as default method.
     if (isset($form['registration_cancellation']['user_cancel_method'][$element]['#access'])) {
-      $form['registration_cancellation']['user_cancel_method'][$element]['#access'] = FALSE;
-    }
-    // Remove the description (only displayed on the confirmation form).
-    else {
-      unset($form['registration_cancellation']['user_cancel_method'][$element]['#description']);
```

Comment:
This line was doing absolutely nothing before, as `'#access'` was not passed from `user_cancel_methods()` before (see https://github.com/backdrop/backdrop-issues/issues/5121). Perhaps what I'm doing here should be moved to a separate PR, after we've made a decision re whether to be hiding such options, or showing them with disabled/locked radios instead (which is what I think provides better UX).

...the code change is rather small though, so if we come to a consensus in https://github.com/backdrop/backdrop-issues/issues/5121, we might as well fix it here.